tests/ui/traits/copy-impl-cannot-normalize.stderr

```diff
@@ -1,9 +1,16 @@
 error[E0277]: the trait bound `T: TraitFoo` is not satisfied
-  --> $DIR/copy-impl-cannot-normalize.rs:22:18
+  --> $DIR/copy-impl-cannot-normalize.rs:22:9
    |
 LL | impl<T> Copy for Foo<T> {}
-   |                  ^^^^^^ the trait `TraitFoo` is not implemented for `T`
+   |         ^^^^ the trait `TraitFoo` is not implemented for `T`
```

Comment:
this span looks worse I think? It feels like we should keep pointing to the self type here :thinking:

Reply:
It's because wfcheck spans were bad, and we were falling back to wfcheck here since we delayed a bug. #106785 should address this.